### ISSUE TITLE
Frontmatter: Cover img not working with transformer-remark

### ISSUE THREAD

**Issue by jonmccon** [Vagr9K/gatsby-advanced-starter#78]
`ERROR #85922  GRAPHQL
There was an error in your GraphQL query:
Field "cover" must not have a selection since type "String" has no subfields.`

Getting this nasty error when trying to pull the cover image with the graphql query
`cover {
          childImageSharp {
            fluid(maxWidth: 1024) {
              ...GatsbyImageSharpFluid
            }
          }
        }`

Followed all the steps outlined by others that had a similar issue, that was solved with using a relative image setup in the config. But still not compiling. https://theleakycauldronblog.com/blog/problems-with-gatsby-image-and-their-workarounds/

I noticed you don't actually call the images in your demo posts, is this functionality that works in other cases? Any insight is helpful, thank you.

**Comment by Vagr9K** [comment]
> I noticed you don't actually call the images in your demo posts, is this functionality that works in other cases?

There are actually examples in the `Big Test` post. You can use markdown combined with the `static` folder path to show optimized images. For example:

```
![Logo Image](logsos/logo-1024.png)
```

But in case of direct image imports via GraphQL, it will depend on user's environment and setup.
The `static` folder is available via `gatsby-source-filesystem`, maybe your image path is outside of it hence the issues?

Try debugging the available GraphQL nodes via GraphiQL. That's probably the best starting point.

**Comment by mhatvan** [comment]
@jonmccon did you manage to solve this in the meantime? I am running into exactly the same issue, trying to figure out how to lazy-load images...

**Comment by jonmccon** [comment]
I was able to reference images in the frontmatter with `featuredImage: ./imageName.png` and querying it with `featuredImage {
          childImageSharp {
            fluid {
              ...GatsbyImageSharpFluid
            }
          }
`
This allows for lazy loading via gatsby-image. I had put my src url in quotes and didn't understand that the error was telling me exactly that. 

However my solution is in conflist with the additions in #58 to the SEO plugin. I had to comment out the image regex that was added, probably has to do with the string formatting. Not sure which is the best way to handle this.
